### PR TITLE
Design show cupons

### PR DESCRIPTION
**Pull request by oscaregr**
Add design for the view where the user see the cupon

# Description

- Design the view for cupons.
- Design responsive.

- What?: I added design to display coupons.
- Why?: It was made just to add the values obtained from DB.

## Screenshots

Captura de pantalla -2023-03-15 17-44-56

### CODE REVIEW

**Review by arthurchina92 — APPROVED** [review]

**Review by Reiyirak — APPROVED** [review]
LGTM

**Review by MaxRetana — APPROVED** [review]
Looks nice!!!

File: app/assets/stylesheets/pages/_showCupons.scss [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+          p {
+            opacity: 90%;
+            padding: .8rem;
+          }
+        }
+      }
+    }
+  }
+}
+
```
Comment (JuanequeX) [inline app/assets/stylesheets/pages/_showCupons.scss:125]: flawless

File: app/controllers/cupons_controller.rb [hunk 2]
```diff
@@ -2,7 +2,9 @@ class CuponsController < ApplicationController
   before_action :authenticate_user!, only:[:new, :create, :destroy]
   def index; end
 
-  def show; end
+  def show
+    # @cupon = cupon
```
Comment (JuanequeX) [inline app/controllers/cupons_controller.rb:6]: Delete this comment

**Review by JuanequeX — CHANGES_REQUESTED** [review]

File: app/controllers/cupons_controller.rb (hunk 2 above)
Comment (oscaregr) [reply app/controllers/cupons_controller.rb:6]: Ok

**Review by JuanequeX — APPROVED** [review]

**Pull request merged**